geckomat/utilities/ecFSEOF/run_ecFSEOF.m

```diff
@@ -0,0 +1,69 @@
+function results = run_ecFSEOF(model,rxnTarget,cSource,alphaLims,Nsteps,file1,file2)
+% 
+% run_ecFSEOF
+%
+% 	Function that runs Flux-scanning with Enforced Objective Function
+% 	for a specified production target.
+%   
+%		model     (struct) ecModel with total protein pool constraint
+%		rxnTarget (string) Rxn ID for the production target reaction, 
+% 				  a exchange reaction is recommended.
+%		cSource	  (string) Rxn name for the main carbon source uptake 
+%			      reaction
+% 		alphaLims (vector) Minimum and maximum biomass yield [gDw/mmol Csource] 
+%				  for enforced objective limits
+%       Nsteps    (integer) Number of steps for suboptimal objective 
+%                 in FSEOF
+%		file1 	  (string) opt, file name for an output .txt tab-separated 
+% 				  file for the results at the genes level
+%		file2 	  (string) opt, file name for an output .txt tab-separated 
+% 				  file for the results at the reactions level		
+%
+% Usage: run_ecFSEOF(ecModel,rxnTarget,cSource,alphaLims,Nsteps)
```

Comment:
Although the inputs are all clearly explained, the output structure is not, so I would recommend including a summary of what does each field in the `results` structure represent.